### PR TITLE
Issue #7580: Updated docs of EmptyBlock

### PR DESCRIPTION
**Pull request by ImmortalRabbit**
Issue #7580 : update doc for EmptyBlock

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 1]
```diff
@@ -103,6 +103,24 @@
  * &lt;module name="EmptyBlock"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private void emptyLoop(){
+ *     for(int i=0; i&lt;10; i++){
```
Comment (esilkensen) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:111]: minor
```suggestion
 *   private void emptyLoop() {
 *     for (int i = 0; i &lt; 10; i++) {
```

**Comment by muhlba91** [comment]
apart from @esilkensen's comment, it looks good to me. please ping if another approval is needed ocne resolved.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 2]
```diff
@@ -103,6 +103,24 @@
  * &lt;module name="EmptyBlock"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private void emptyLoop() {
+ *     for (int i = 0; i &lt; 10; i++) {
+ *       // violation
```
Comment (rdiachenko) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:112]: This comment should be on the line where violation occurred. Please fix all below cases as well.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 3]
```diff
@@ -111,7 +129,24 @@
...
+ *       // ignored
+ *     }
  *
+ *     try {
+ *       // violation
+ *     } catch (Exception e) {
+ *       // ignored
+ *     }
+ *   }
+ * }
```
Comment (rdiachenko) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:148]:
```
/var/tmp/checkstyle $ cat Test.java 
public class Test {
  private void emptyLoop() {
    for (int i = 0; i < 10; i++) {
      // ignored
    }

    try {
      // violation
    } catch (Exception e) {
      // ignored
    }
  }
}
```
```
/var/tmp/checkstyle $ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
       <module name="EmptyBlock">
         <property name="option" value="text"/>
         <property name="tokens" value="LITERAL_TRY"/>
       </module>
    </module>
</module>
```
```
/var/tmp/checkstyle $ java -jar checkstyle-8.30-all.jar -c config.xml Test.java 
Starting audit...
Audit done.
```
In your example you say there is a violation but it is not true.

**Comment by rdiachenko** [comment]
@ImmortalRabbit ping

**Comment by ImmortalRabbit** [comment]
> @ImmortalRabbit ping

sorry, I am little bit busy with my final exams, will come back to fix suggestions in one or two days.

**Comment by romani** [comment]
rebased

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java (hunk 2 above)
Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java]: fixed

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java (hunk 3 above)
Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java]: fixed

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
Github, generate web site

**Pull request merged**